examples/mobile/UIComponents/components/list/icon-text-list.html

```diff
@@ -26,7 +26,7 @@
 			<ul class="ui-listview ui-content-area">
 				<li class="ui-li-has-icon ui-li-divider">
 					<div class="ui-li-icon">
-						<img src="images/thumbnail.jpg"/>
+						<img src="images/tw_list_icon_connections.svg"/>
```

Comment:
In the Guide, the icons also have the each colors like "#53a4ff" etc.
Can you apply the color?

Reply:
I applied a color directly to the svg file.
```
<svg .... fill="#53a4ff">
```